README.md

```diff
@@ -14,11 +14,19 @@ It enables you to query package sizes.
 
 ## Installation
 
+*Installation is not necessary if you use [npx] \(see [Usage](#usage)).*
```

Comment:
I would rather make a statement, like
> You can either install globally `bundle-phobia-cli` utils or invoke it on the fly using npx [see usage doc...]

adding below as example the both commands `npm install -g` and the `npx -p`